src/library/scala/collection/mutable/HashTable.scala

```diff
@@ -429,16 +429,7 @@ private[collection] object HashTable {
   /**
    * Returns a power of two >= `target`.
    */
-  private[collection] def powerOfTwo(target: Int): Int = {
-    /* See http://bits.stephan-brumme.com/roundUpToNextPowerOfTwo.html */
-    var c = target - 1
-    c |= c >>>  1
-    c |= c >>>  2
-    c |= c >>>  4
-    c |= c >>>  8
-    c |= c >>> 16
-    c + 1
-  }
+  private[collection] def nextPositivePowerOfTwo(target : Int): Int = 1 << -numberOfLeadingZeros(target - 1)
```

Comment:
should probably be extracted in a static method somewhere outside instead, it's not really strictly to maps, rather integers.